### PR TITLE
[NDS-861] feat(Menu): new menu component

### PR DESCRIPTION
**Pull request by panvourtsis**
## Description

This PR introduce the new menu component and replace the old from topbar

[Ticket](https://www.figma.com/file/8kMPBNYHHXz2AtkzeeDmk5/DS-%7C-V5-%5BWIP%5D?type=design&node-id=23277-132174&mode=design&t=IcWAchvI7iM86IBc-0)

### CODE REVIEW

File: src/components/Menu/Menu.tsx [hunk 1]
```diff
@@ -1,106 +1,90 @@
...
-            >
-              <span>{buttonText}</span>
-            </Button>
-          }
+    <MenuTrigger isOpen={isOpen} onOpenChange={onClose}>
+      <Popover
+        triggerRef={triggerRef}
+        css={css`
+          min-width: 150px;
+          background: #ffffff;
```
Comment (kostasdano) [inline src/components/Menu/Menu.tsx:54]: maybe move this to Menu.style.ts and use rem for min-width and global colors (or ideally menu tokens) for the background

File: src/components/Menu/Menu.style.ts [hunk 2]
```diff
@@ -1,29 +1,84 @@
...
+
+export const MenuItemWrapper = styled(MenuItem)<{
+  isDisabled?: boolean;
+  isCompact?: boolean;
+  rowSize?: ListRowSize;
+}>(({ rowSize, isCompact, isDisabled, theme }) => {
+  const listItemTokens = getListItemTokens(theme);
+  const height = rem(LIST_ITEM_HEIGHT[rowSize]);
+  const padding = css`0 ${listItemTokens('paddingHorizontal')}`;
+  const itemTypographyStyle = isCompact ? body03(theme) : body02(theme);
```
Comment (kostasdano) [inline src/components/Menu/Menu.style.ts:20]: are there menu tokens for these maybe?

File: src/components/Menu/Menu.style.ts [hunk 3]
```diff
@@ -1,29 +1,84 @@
...
+    background-color: ${listItemTokens('backgroundColor.default')};
+
+    & > div {
+      ${itemTypographyStyle}
+    }
+
+    div[parenttype='Menu'] {
+      display: flex;
+      flex-direction: row;
+      gap: ${rem(12)};
```
Comment (kostasdano) [inline src/components/Menu/Menu.style.ts:32]: same

**Review by kostasdano — APPROVED** [review]

File: src/components/Menu/Menu.style.ts (hunk 2 above)
Comment (panvourtsis) [reply src/components/Menu/Menu.style.ts:20]: MenuItemWrapper is just a wrapper of listitems, these are the tokens for them

File: src/components/Menu/Menu.tsx (hunk 1 above)
Comment (panvourtsis) [reply src/components/Menu/Menu.tsx]: fixed https://github.com/Orfium/orfium-ictinus/pull/747/commits/dd9af368f8341e0533aca239112f1d4e39a07471

**Pull request merged**